plenum/test/monitoring/test_throughput_based_master_degradation_detection.py

```diff
@@ -278,3 +285,15 @@ def test_master_not_degraded_on_revival_spike_on_one_backup_while_load_stopped(t
     throughput_ratio = get_throughput_ratio(inst_req_streams, tconf)
 
     assert_master_not_degraded(throughput_ratio, tconf)
+
+
+def test_master_not_degraded_on_new_instance(fake_monitor, tconf):
```

Comment:
The test rather checks that newly added instance is not degraded.
I think we need two tests: that master is not degraded when a new instance is added, and that new instance is not degraded right after it's added.

Reply:
May be I can add a new check for master instance in this test or we need two tests with different checkings for one case?